.github/workflows/build.yml

```diff
@@ -25,4 +25,22 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         run: |
           echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-          docker push antrea/ipfix-collector:latest
\ No newline at end of file
+          docker push antrea/ipfix-collector:latest
+  upload-kafka-images:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Pull, tag and push confluentinc/cp-zookeeper and confluentinc/cp-kafka Docker images to Antrea Docker registry
+        if: ${{ github.repository == 'vmware/go-ipfix' && github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        run: |
+          echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+          docker pull confluentinc/cp-zookeeper:latest
+          docker tag confluentinc/cp-zookeeper:latest antrea/kafka-zookeeper:latest
+          docker push antrea/kafka-zookeeper:latest
```

Comment:
I will request Antonin for permission to upload these images on to Antrea repo on dockerhub.

Reply:
Permissions are provided for these images: `antrea/confluentinc-zookeeper` and `antrea/confluentinc-kafka`

Reply:
I uploaded following images directly using goipfixbot docker login onto Antrea repo:
`antrea/confluentinc-zookeeper:6.2.0` and `antrea/confluentinc-kafka:6.2.0`

We do not need any github workflow to upload images. We use this versioned image as long as it works for us. When we have to move to a new one, we directly upload the new one. 
Please edit the manifest with the above tags and remove the GitHub workflow. Please add this info in README if possible.